services/QuillLMS/app/lib/quill_authentication.rb

```diff
@@ -68,10 +70,22 @@ def sign_in(user)
     @current_user = user
   end
 
+  def demo_id=(demo_id)
```

Comment:
What about `current_user_demo_id` instead of `demo_id`? This method modifies `@current user`, but the naming indicates that it is only modifying an instance variable.